node/src/actors/chain_manager/mod.rs

```diff
@@ -1657,7 +1668,7 @@ mod tests {
             chain_manager.get_superblock_beacon(),
             CheckpointBeacon {
                 checkpoint: 0,
-                hash_prev_block: genesis_hash,
+                hash_prev_block: Hash::default(),
```

Comment:
I think this test is supposed to check that the superblock genesis hash is set correctly. And it is not, because the `SuperBlockState` is not initialized. You could try to do `chain_manager.chain_state.superblock_state = SuperBlockState::new(...)` and set the genesis hash there.